src/DynamoCore/DynamoCore.csproj

```diff
@@ -85,6 +85,7 @@ limitations under the License.
       <HintPath>..\..\extern\Newtonsoft.json\Newtonsoft.Json.dll</HintPath>
```

Comment:
Hi @aosyatnik, these assemblies cannot be added to `DynamoCore` because `DynamoCore` can now run on Linux (which does not contain these Windows specific assemblies). Please have a word with @vmoyseenko, he should be able to tell you more :)